examples/benchmark/main.zig

```diff
@@ -68,7 +68,7 @@ pub fn asyncMain() !void {
                         deviceKind,
                     });
                     // we only list 1 CPU device
-                    if (target == .cpu) break;
+                    if (target == .cpu and !platform.compilation_options.sharding_enabled) break;
```

Comment:
can you use `platform.sharding().num_partitions > 1` here too ? We want to be able to customize the logic.

Reply:
Done